external-builds/pytorch/build_prod_wheels.py

```diff
@@ -602,9 +602,31 @@ def do_build_pytorch(
     pytorch_build_version = (pytorch_dir / "version.txt").read_text().strip()
     pytorch_build_version += args.version_suffix
     print(f"  Default PYTORCH_BUILD_VERSION: {pytorch_build_version}")
-    print(
-        f"  Flash attention enabled: {args.enable_pytorch_flash_attention_windows or not is_windows}"
-    )
+
+    # Enable/disable only for non-Windows
+    if not is_windows:
+        if "2.7" in str(pytorch_build_version):
+            env["USE_FBGEMM_GENAI"] = "ON"
+            use_flash_attention = (
+                "1" if args.enable_pytorch_flash_attention_linux else "0"
+            )
+            env.update(
+                {
+                    "USE_FLASH_ATTENTION": use_flash_attention,
+                    "USE_MEM_EFF_ATTENTION": use_flash_attention,
+                }
+            )
+            print("FBGEMM_GENAI and Flash Attention enabled (PyTorch 2.7, Linux)")
```

Comment:
This log line doesn't consider the value of `use_flash_attention`. It will say "enabled" when it might be disabled.